File: LeetCode/Easy/1051-height-checker/1051-height-checker.go
func heightChecker(heights []int) int {
    ret := 0
    compare := make([]int, len(heights))
    copy(compare, heights)
    sort.Ints(compare)
    
    for i := 0; i < len(heights); i++ {
        if heights[i] != compare[i] { ret++ }
    }
    
    return ret
}

File: LeetCode/Easy/1051-height-checker/1051-height-checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        int ret = 0;
        vector<int> compare = heights;
        sort(compare.begin(), compare.end());
        
        
        for (int i = 0; i < heights.size(); ++i)
            if (heights[i] != compare[i]) ++ret;
        
        return ret;
    }
};